crates/assists/src/handlers/remove_dbg.rs

```diff
@@ -49,14 +49,28 @@ fn adjusted_macro_contents(macro_call: &ast::MacroCall) -> Option<String> {
         macro_text_with_brackets.len() - TextSize::of(')'),
     ));
 
-    let is_leaf = macro_call.syntax().next_sibling().is_none();
+    let is_leaf = is_leaf_or_control_flow_expr(macro_call);
```

Comment:
Nit: the variable name now is a bit wrong.

IMO we can just inline it, it was extracted by me initially to document what `macro_call().syntax().next_sibling().is_none()` had meant, but now with a properly named function introduced it makes no sense anymore.